.gitignore

```
@@ -1,4 +1,3 @@
 target
 *.bk
-.idea
-Cargo.lock
+.idea
```

Kep the last empty line
```suggestion
.idea

```